application/controllers/chat.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+session_start();
```

Comment:
Is there any rule that session_start() needs to happen at the top of the file?

Reply:
Sir in this file I am just checking whether the user is valid or not like if case user is not valid and try to enter the page just type the URL in the address bar, in that case, I use the session to verify the details if a user is not valid I immediately destroy the session and redirect to the login in page.